jbosstools/multiple/jbosstools-multiple.target

```diff
@@ -24,6 +24,7 @@
       <unit id="org.apache.commons.lang3" version="3.1.0.v201403281430"/>
       <unit id="org.apache.commons.logging" version="1.2.0.v20180409-1502"/>
       <unit id="org.apache.httpcomponents.httpcore" version="4.4.14.v20210128-2225"/>
+      <unit id="org.apache.httpcomponents.httpclient" version="4.5.13.v20210128-2225"/>
```

Comment:
Not need to update httpcore as well ?

Reply:
there is only one version available on orbit for httpcore for 2021-03.